manifests/init.pp

```diff
@@ -169,17 +169,19 @@
   Boolean                         $manage_root_metadata       = true,
   Boolean                         $manage_root_perms          = true,
   Boolean                         $manage_rc_local            = true,
-  Boolean                         $pam                        = simplib::lookup('simp_options::pam', { 'default_value'   => false }),
-  Boolean                         $ldap                       = simplib::lookup('simp_options::ldap', { 'default_value'  => false }),
-  Boolean                         $sssd                       = simplib::lookup('simp_options::sssd', { 'default_value'  => true }),
+  Boolean                         $pam                        = simplib::lookup('simp_options::pam',  { 'default_value' => false }),
+  Boolean                         $ldap                       = simplib::lookup('simp_options::ldap', { 'default_value' => false }),
+  Boolean                         $sssd                       = simplib::lookup('simp_options::sssd', { 'default_value' => true }),
   Boolean                         $stock_sssd                 = true,
   Boolean                         $classification_warning     = true,
   String                          $vardir_owner,
   String                          $vardir_group,
   String                          $vardir_mode,
 ) {
+  # Bolt sets the environment to be 'bolt_catalog', and it behaves a differently than SIMP typically assumes
+  $running_in_bolt = $environment == 'bolt_catalog'
```

Comment:
This makes sense but shouldn't it go into `simplib` or `stdlib` as a function or a fact?

Reply:
Discussion on https://github.com/simp/pupmod-simp-simp/pull/188